utils/rpms/daos.spec

```diff
@@ -3,11 +3,11 @@
 %define agent_svc_name daos_agent.service
 
 %global mercury_version 2.0.1~rc1-1%{?dist}
-%global libfabric_version 1.11.1
+%global libfabric_version 1.12.0~rc1-1
```

Comment:
Need to update https://github.com/daos-stack/daos/blob/15fe4f8b7a22168afa319ab5a36605fd00105e09/debian/control#L8 similarly.